ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -8,22 +8,27 @@ import UIKit
 
 class DemoListViewController: UITableViewController {
 
-    static func addDemoListTo(window: UIWindow, pushing viewController: UIViewController?) {
-        if let colorProvider = window as? ColorProviding, let primaryColor = colorProvider.primaryColor(for: window) {
-            Colors.setProvider(provider: colorProvider, for: window)
-            FluentUIFramework.initializeAppearance(with: primaryColor, whenContainedInInstancesOf: [type(of: window)])
-        } else {
-            FluentUIFramework.initializeAppearance(with: Colors.primary(for: window))
-        }
+    private var provider: ColorProviding? = DemoColorTheme.default.provider
+    public var themeName: String = DemoColorTheme.default.name
```

Comment:
Any reason this needs to be public? It'd be great if we could make this private.

Reply:
It's public because it's accessed/set in the ColorDemoController when the theme is changed